iceoryx_utils/include/iceoryx_utils/internal/relocatable_pointer/relative_ptr.hpp

```diff
@@ -234,13 +234,13 @@ class relative_ptr : public RelativePointer
 
 
     relative_ptr(const RelativePointer& other)
+        : RelativePointer(other)
     {
-        m_offset = computeOffset(other.computeRawPtr());
     }
 
     relative_ptr& operator=(const RelativePointer& other)
     {
-        m_offset = computeOffset(other.computeRawPtr());
+        RelativePointer::operator=(other);
```

Comment:
doesn't `*this = other` also work?

Reply:
Yes, it does work as well. You prefer `*this`?

Reply:
It's more idiomatic ... at least I've seen it way more than the explicit call of the assignment operator

Reply:
Yes it is more idiomatic. For operators I usually use ``rhs`` instead of ``other``, I wonder why I did not do so in the past.
You will also sometimes find ``that`` instead of ``other``, but this is just a matter of style and convention.

Historically it is a bit of prototype code and could be cleaned up, but there seemed to be no glaring bugs (that I know of).

Reply:
Done.

Reply:
> doesn't `*this = other` also work?

Of course this doesn't work (infinite recursion)

Reply:
Fixed.